### PR TITLE
feat(Landmark): Change CardSide attribute to flipped (boolean)

### PR DESCRIPTION
**Pull request by keytsai**
#81 
1. Landmark 的 CardSide 屬性改為 flipped (boolean)
2. 將 `ownedEstablishment` 重新命名為 `establishments`
3. 將 `ownedLandmark`  重新命名為 `landmarks`

### CODE REVIEW

File: src/main/java/domain/card/establishment/FruitAndVegetableMarket.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public FruitAndVegetableMarket() {
     protected void doTakeEffect(Game game) {
         // 當你自己骰出這個數字時，每擁有一張[CardType.CROP]的建築，就可以從銀行獲得2元。
         if (isTurnPlayer(game, getOwner())) {
-            var cropCardCount = (int) getOwner().getOwnedEstablishment().stream()
+            var cropCardCount = (int) getOwner().getEstablishments().stream()
                     .filter(establishment -> establishment.getCardType().equals(CardType.CROP))
                     .count();
```
Comment (shmily7829) [inline src/main/java/domain/card/establishment/FruitAndVegetableMarket.java:20]: 這段看能不能把篩選邏輯封裝在 `getEstablishments` 
```java
getOwner().getEstablishments(CardType.CROP)
```

**Review by shmily7829 — COMMENTED** [review]
Review done.

File: src/test/java/domain/PlayerTest.java [hunk 2]
```diff
@@ -56,15 +56,15 @@ void flipBackLandMark() {
 
         //then
         assertThat(player.getTotalCoin()).isEqualTo(originalBalanceOfPlayer - 16);
-        assertThat(player.getOwnedLandmark().get(2).getCardSide()).isEqualTo(Landmark.CardSide.FRONT);
+        assertThat(player.getLandmarks().get(2).isFlipped()).isTrue();
```
[the remark this answers is not in the document]
Comment (keytsai) [reply src/test/java/domain/PlayerTest.java]: Done

File: src/test/java/domain/PlayerTest.java [hunk 3]
```diff
@@ -56,15 +56,15 @@ void flipBackLandMark() {
...
     }
 
     @Test
     void flipFrontLandMark() {
         //given 玩家有正面的主題樂園
         var originalBalanceOfPlayer = player.getTotalCoin();
         var amusementPark = new AmusementPark();
-        player.getOwnedLandmark().get(2).setCardSide(Landmark.CardSide.FRONT);
+        player.getLandmarks().get(2).setFlipped(true);
```
[the remark this answers is not in the document]
Comment (keytsai) [reply src/test/java/domain/PlayerTest.java]: Done

File: src/main/java/domain/card/establishment/FruitAndVegetableMarket.java (hunk 1 above)
Comment (keytsai) [reply src/main/java/domain/card/establishment/FruitAndVegetableMarket.java]: Done

**Review by shmily7829 — APPROVED** [review]
LGTM.

**Pull request merged**